pkg/store/sandbox/sandbox.go

```diff
@@ -58,6 +62,11 @@ func (s *Store) Add(sb Sandbox) error {
 	if _, ok := s.sandboxes[sb.ID]; ok {
 		return store.ErrAlreadyExist
 	}
+
```

Comment:
nit: empty line not needed and the following one.

Reply:
done
Are there rules?  @Random-Liu

Reply:
I was told that we should avoid unnecessary empty lines to make the code short and easy to read. So that we could have more code in one screen and easier for reader to understand.

Some convention inside Google. :)

Reply:
Not from google but i hate empty lines as well ;)

Reply:
@crosbymichael OK. So everyone hates empty lines not only Google. :)

Reply:
The rules are subjective:)